### PR TITLE
Add spark-rapids pulls to GitHub project [skip ci]

### PR DESCRIPTION
**Pull request by mattahrens**
Adding new PRs to global spark-rapids GitHub project.  Uses same action as what we put in for issues.

### CODE REVIEW

**Comment by mattahrens** [comment]
build

File: .github/workflows/add-pr-to-project.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Add new pull requests to project
+
+on:
+  pull_request:
```
Comment (pxLi) [inline .github/workflows/add-pr-to-project.yml:18]: plz try: `pull_request_target`

`pull_request` from forked repo should not have access to prod secret

**Review by pxLi — CHANGES_REQUESTED** [review]
Please add the trigger event `pull_request_target` to the existing workflow https://github.com/NVIDIA/spark-rapids/blob/branch-22.12/.github/workflows/add-to-project.yml#L20

e.g.
```
on:
  issues:
    types:
      - opened
  pull_request_target:
    types:
      - opened
```

**Comment by mattahrens** [comment]
Thanks for the feedback.  I moved the pull request addition to the existing workflow.

**Comment by mattahrens** [comment]
build

**Review by pxLi — APPROVED** [review]

**Pull request merged**